### PR TITLE
🌱 Release v1.4.9

### PR DESCRIPTION
**Pull request by g-gaston**
/area release

### CODE REVIEW

**Review by CecileRobertMichon — COMMENTED** [review]
/approve
/hold

**Review by furkatgofurov7 — APPROVED** [review]
/approve

Unless there is no diff between now and the time we merge this PR

File: CHANGELOG/v1.4.9.md
Comment (sbueringer) [inline CHANGELOG/v1.4.9.md:14]: Sorry one PR to add: https://github.com/kubernetes-sigs/cluster-api/pull/9809

(cherry-pick didn't make it before because of merge conflicts)

**Review by furkatgofurov7 — COMMENTED** [review]
/lgtm

**Pull request merged**
